src/System.Data.Common/src/System/Data/DataTable.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+
+namespace System.Data {
+   
+    // An empty Datatable class to allow complete interfaces
+    public class DataTable{
+
```

Comment:
Is it intentional to have an internal constructor in the reference assembly but a public constructor here in the implementation? (By not specifying a constructor here, the C# compiler will generate a public default constructor).